katello/hooks/pre/32-cdn_setting.rb

```diff
@@ -0,0 +1,10 @@
+if module_enabled?('katello')
+    if param_value('katello', 'cdn_ssl_version')
+      param = param_value('katello', 'cdn_ssl_version')
+      logger.info 'cdn_ssl_version param found, migrating to a Katello setting'
+      `foreman-rake -- config -k cdn_ssl_version -v #{param}`
```

Comment:
In the pre hook I'm not sure you can rely on the setting being present in an upgrade scenario. The database might still need a migration and seeding. Is there are reason you're doing this in a pre job rather than doing it all in the post hook?